package.json

```diff
@@ -22,10 +22,6 @@
     "concurrently": "^7.6.0",
     "prettier": "^2.8.3"
   },
-  "workspaces": [
```

Comment:
How come this gets removed? I thought it would be "packages/*"

Reply:
this is only used for yarn and npm. pnpm uses the `pnpm-workspaces.yaml` file